pages/dkp/kommander/1.4/install/index.md

```diff
@@ -29,3 +29,8 @@ You can also reach the Kommander UI via this url, given that Kommander is deploy
 ```
 https://<CLUSTER_URL>/ops/portal/kommander/ui
 ```
+
+[konvoy-base-addons]: /dkp/konvoy/1.8/addons/
+[konvoy-download]: /dkp/konvoy/1.8/download/
+[konvoy-install]: /dkp/konvoy/1.8/quick-start/#installing-konvoy
+[konvoy-prereq]: /dkp/konvoy/1.8/quick-start/#prequisites
```

Comment:
```suggestion
[konvoy-prereq]: /dkp/konvoy/1.8/quick-start/#prerequisites
```
kept staring at this word feeling like something was wrong then finally saw it 😆

Reply:
OH MY! Good catch